packages/SwingSet/src/kernel/loadVat.js

```diff
@@ -195,7 +195,14 @@ export function makeVatLoader(stuff) {
       );
     }
 
-    kernelSlog.addVat(vatID, isDynamic, description, name, vatSourceBundle);
+    kernelSlog.addVat(
+      vatID,
+      isDynamic,
+      description,
+      name,
+      vatSourceBundle,
+      managerType,
+    );
```

Comment:
How did you discover that we missed this?

It seems like something that static typing should have caught. I don't suppose these correspond exactly to `ManagerOptions` but is it worth putting some part of these args into an object with named properties and giving them a type? Perhaps not, but it's something to think about...

Reply:
This was merely in the slogger, which doesn't take the full ManagerOptions object. And as a diagnostic thing, it's pretty low-priority for adding typechecking..